pymc/model.py

```diff
@@ -920,111 +927,112 @@ def test_point(self) -> Dict[str, np.ndarray]:
     @property
     def initial_point(self) -> Dict[str, np.ndarray]:
         """Maps free variable names to transformed, numeric initial values."""
-        if set(self._initial_point_cache) != {get_var_name(k) for k in self.initial_values}:
-            return self.recompute_initial_point()
-        return self._initial_point_cache
+        seed = self.rng_seeder.randint(2 ** 30, dtype=np.int64)
+        return self.recompute_initial_point(seed)
 
-    def recompute_initial_point(self) -> Dict[str, np.ndarray]:
-        """Recomputes numeric initial values for all free model variables.
+    def recompute_initial_point(self, seed) -> Dict[str, np.ndarray]:
```

Comment:
I forgot, what's the point of compute and recompute methods, if we are not caching the results?